tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3680,4 +3680,35 @@ public function userDownloadsTheSpaceUsingTheWebdavApi(string $user, string $spa
 			)
 		);
 	}
+
+	/**
+	 * @Given user :sharer has shared resource :path inside space :space with user :sharee
+	 *
+	 * @param $sharer string
+	 * @param $path string
+	 * @param $space string
+	 * @param $sharee string
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function userHasSharedFileInsideSpaceWithUser(string $sharer, string $path, string $space, string $sharee): void {
+		$sharer = $this->featureContext->getActualUsername($sharer);
+		WebDavHelper::$SPACE_ID_FROM_OCIS = $this->getSpaceIdByName($sharer, $space);
```

Comment:
we need to adjust `SPACE_ID_FROM_OCIS` this variable as this seems a bit odd since this is the `ocis`. This will also be the refactor, may be later. @saw-jan @SwikritiT @amrita-shrestha @grgprarup

Reply:
I actually forgot to remove the line. :laughing:  Thanks for noticing. :+1: